doc/content/getting-started/programming-software/st-examples/stcubeide/_index.md

```diff
@@ -7,10 +7,26 @@ distributions: null
 
 If you are using [STM32CubeIDE](https://www.st.com/en/development-tools/stm32cubeide.html) as your build environment, then using the same tool to program your {{% gnse %}} is convenient.
 
-<!--more-->
+{{< figure src="stm32cubeide.png" alt="stm32 cube ide" >}}
 
-## Steps to reproduce
+## Running Project
 
-1. To program and run or debug {{% gnse %}}, right-click on it and select ***Run As*** or ***Debug As***. When programming, make sure you select **STM32 Cortex-M C/C++ Application** in the **Run As** or **Debug As** section.
+When the [build]({{< ref "/getting-started/se-sw/env-setup/stm32cubeide" >}}) process is done you can flash the binary file to the {{% gnse %}} by running the program from the STM32CubeIDE.
 
-{{< figure src="cubeide.png" alt="cubeide run as" >}}
+Right click on the project name in the **Project Explorer** and select **Run As -> STM32 Cortex-M C/C++ Application**.
+
+{{< figure src="run_as.png" alt="project run as" >}}
+
+This will start to **flash** the compiled program (bin file) to the {{% gnse %}}. The following figure shows the **Console** output, for example, when running the [basic]({{< ref "/applications/se-basic" >}}) project.
```

Comment:
This actually flashes the .elf file as it uses GDB server to do so.
I think there  is no need to mention the file type.

![image](https://user-images.githubusercontent.com/24204263/133120309-95217f8e-ccb1-4097-b3d7-053187f64b1f.png)

Reply:
Thanks for the clarification. I thought it was the bin file.
fixed